exercises/custom-set/custom_set_test.go

```diff
@@ -6,20 +6,16 @@ package stringset
 //
 // New() Set
 // NewFromSlice([]string) Set
-// (s Set) Add(string)         // modify s
-// (s Set) Delete(string)      // modify s
-// (s Set) Has(string) bool
-// (s Set) IsEmpty() bool
-// (s Set) Len() int
-// (s Set) Slice() []string
 // (s Set) String() string
-// Equal(s1, s2 Set) bool
-// Subset(s1, s2 Set) bool     // return s1 ⊆ s2
+// (s Set) IsEmpty() bool
+// (s Set) Has(string) bool
+// (s Set) Add(string)
```

Comment:
Should these be listed in the same order they are tested? I think that was what you were going for. In that case, let's move this one down.